WordPress/src/main/res/values/strings.xml

```diff
@@ -2536,25 +2540,25 @@
     <string name="app_rating_rate_never">No thanks</string>
 
     <!-- Gutenberg mobile strings -->
-    <string tools:ignore="UnusedResources" name="gutenberg_mobile_gutenberg_packages_format_library_src_link_modal_native_js_101">Link inserted</string>
```

Comment:
I haven't checked these in detail as I assume it's just a result of the auto-formatter, right?

Reply:
Yep!